lib/state/appStore.ts

```diff
@@ -195,12 +195,15 @@ export const useAppStore = create<AppState>()((set, get) => {
       ? selectedWalletId
       : 0;
 
+  const fiatKey = secureStorage.getItem(fiatCurrencyKey);
+  const fiatCurrency = fiatKey === "" ? "" : fiatKey || "USD";
```

Comment:
`const fiatCurrency = secureStorage.getItem(fiatCurrencyKey) ?? "USD";` might work?

Reply:
True, I was unsure about this and went with the safe option to change it later, done now!